tests/integration/linux_bridge_test.py

```diff
@@ -189,7 +211,26 @@ def test_add_port_to_existing_bridge(bridge0_with_port0, port1_up):
     assertlib.assert_state(desired_state)
 
 
-def test_linux_bridge_uses_the_port_mac(port0_up, bridge0_with_port0):
+@pytest.mark.xfail(
```

Comment:
My question in https://github.com/nmstate/nmstate/pull/543#discussion_r345322153 is still open, not sure why GitHub fails to keep it here.